### PR TITLE
Updates our credits.

### PR DESCRIPTION
**Pull request by yoyobatty**
No CL needed.

I hope I got everyone.

### CODE REVIEW

**Review by ghost — APPROVED** [review]
Brings a tear to my eye...

If you missed someone we can simply re-open the PR.

File: html/changelog.html [hunk 1]
```diff
@@ -35,12 +35,12 @@
...
-			<font size='2'><b>Host:</b> ErikHanson & AdmiralHippie<br></font>
-			<font size='2'><b>Head Admins:</b> DudeInYourBackyard, Carbonhell, tfw</font><br>
-			<font size='2'><b>Head Coders:</b> Pyko & ErikHanson <br></font>
-			<font size='2'><b>Game Masters:</b> BardOfRage, TeamStab, 2dollarbill, Looping, fluxcapacitor1337, tortellinitony, Maroonenahmoron<br></font>
-			<font size='2'><b>Game Admins:</b> Bartels, grxx, AbusiveToaster, Slehmerbo, Rikaru19xjenkins, Hackole, Lordkruor, Gandalf2k15, Brawzki, Oprayx73, swagile, Spankmaster, Woross<br></font>
-			<font size='2'><b>Code Maintainers:</b> TDcoolguy300, Pyko, GuyonBroadway, JohnGinnane, ErikHanson, FrozenGuy5, tortellinitony, Banthebantz<br></font>
+			<font size='2'><b>Host:</b> TDcoolguy300 aka Gregg<br></font>
+			<font size='2'><b>Head Admins:</b> LxToofonea, Carbonhell, tfw</font><br>
+			<font size='2'><b>Head Coders:</b> Pyko & YoYoBatty <br></font>
+			<font size='2'><b>Game Masters:</b> Maroonenahmoron, Yskey, Kawaii Big Boss, EagleEyes, TheMeatMan, Xenon, Yobob<br></font>
```
Comment (Manly-Dwarf) [inline html/changelog.html:41]: Who is Kawaii Big Boss?
I only know of a guy named Manly Dwarf.
:smug:

Comment (ghost) [reply html/changelog.html:41]: Who's Manly Dwarf?

**Comment by Jujumatic** [comment]
it seems this got reverted or something cause i looked at the changelog today and for some reason it still mentions erik and hippie

**Pull request merged**